### PR TITLE
Added Avatars badges

### PR DESCRIPTION
**Pull request by sri-rishi**
Avatar types: 

- Image Avatar
- Avatar with Text

Badge types:
- Status Badge
- Badge with Icon
- Text Badge

### CODE REVIEW

File: lib/Components/badges.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    background: var(--red-color);
+}
+
+.status-idle-badge {
+    background: var(--darker-yellow-color);
+}
+
+.badge-container > i {
+    font-size: 2rem;
+}
```
Comment (gauravsinhaweb) [inline lib/Components/badges.css:69]: use shorter classname for short typing

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: lib/Components/badges.css (hunk 1 above)
Comment (sri-rishi) [reply lib/Components/badges.css:69]: But shorter than this will not make sense to user what classes are used for and what are they means

**Comment by Naman-Saxena1** [comment]
I checked the code and preview, can't find any major issues, looks good to me 👍

**Comment by sri-rishi** [comment]
Merged to dev branch

**Pull request merged**